src/components/common/ExpressionInput.vue

```diff
@@ -155,13 +161,44 @@ export class ExpressionInput extends mixins(Lang) {
     ]
   }
 
+  /**
+   * Making sure we port the auto-suggest under desired dom
+   */
+  portAutoSuggestContent() {
+    // override position which came from AutoSuggest original code
+    this.autoSuggestDropdown.style.left = 0
+    this.autoSuggestDropdown.style.top = `${this.$refs.input.clientHeight}px`
+
+    // move the created dropdown suggest inside the desired dom
```

Comment:
- [ ] will correct the syntax to `move the created suggest's dropdown inside the desired dom`